NorthwindCRUD/Filters/EnumSchemaFilter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace NorthwindCRUD.Filters
+{
+    public class EnumSchemaFilter : ISchemaFilter
+    {
+        public void Apply(OpenApiSchema schema, SchemaFilterContext context)
+        {
+            if (context.Type.IsEnum)
```

Comment:
Probably you should a null check or not enum check, something like

 if (context.Type == null || !context.Type.IsEnum) return;

Reply:
It's "void" I don't think return is required. Also context.Type could not be null as it represents the type of the schema being processed.